apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -46,7 +54,21 @@ export default class CourseScript extends Component {
             color={Button.ButtonColor.gray}
           />
         </div>
+        {viewAs === ViewType.Teacher && (
+          <TeacherInfoBox>
+            <div style={{opacity: selectedSectionId ? 1 : 0.3}}>
+              This will be a toggle for hiding/showing the unit, and will be disabled
+              if we have no section selected.
+            </div>
+          </TeacherInfoBox>
+        )}
       </div>
     );
   }
 }
+
+export default connect(state => ({
+  // TODO: move viewAs out of stage lock
```

Comment:
Not sure how easily this will be, but I want to address in a separate PR. I'm not 100% sure I need this (as I could just pass through `isTeacher` as a prop), but we've talked about using the view as teacher/student toggle more broadly, so it does feel like it would be a good thing to separate it from stageLock.